core/services/bulletprooftxmanager/bulletprooftxmanager_test.go

```diff
@@ -36,7 +36,8 @@ func TestBulletproofTxManager_SendEther_DoesNotSendToZero(t *testing.T) {
 	to := utils.ZeroAddress
 	value := assets.NewEth(1)
 
-	_, err := bulletprooftxmanager.SendEther(db, big.NewInt(0), from, to, *value, 21000)
+	q := pg.NewQ(db, logger.NewNullLogger(), cltest.NewTestGeneralConfig(t))
```

Comment:
I think you can reference the `logger.NullLogger` instance instead of making new ones.
Although, why suppress these logs at all? What about `logger.TestLogger(t)` so they can be flipped on for debugging?

Reply:
Fixed